catroid/src/main/java/org/catrobat/catroid/utils/notifications/StatusBarNotificationManager.java

```diff
@@ -119,12 +121,14 @@ public void showOrUpdateNotification(Context context, NotificationData notificat
 		int notificationID = notificationData.getNotificationID();
 		notificationData.setProgressInPercent(progressInPercent);
 		if (progressInPercent < MAX_PERCENT) {
+			ToastUtil.showSuccess(context,notificationData.getTextWorking());
```

Comment:
Please use the features "Reformat Code" and "Optimize Imports" in Android Studio to fix coding style issues and imports. See also [Static Analysis warnings](https://jenkins.catrob.at/job/Catroid/job/PR-4317/10/analysis/#issuesContent)
```suggestion
			ToastUtil.showSuccess(context,notificationData.getTextWorking());
```